### PR TITLE
Add Account and Corpus Management APIs to Playground

### PR DESCRIPTION
**Pull request by pwoznic**

### CODE REVIEW

File: www/static/vectara-oas.yaml [hunk 1]
```diff
@@ -515,6 +571,41 @@ paths:
...
+              $ref: "#/components/schemas/adminReadCorpusRequest"
+        required: true
+      tags:
+        - AdminService
+      parameters:
+        - $ref: '#/components/parameters/customerIdentifier'
+        - $ref: '#/components/parameters/timeout'
+      description: |
+        Displays detailed information about corpora within your account including basic information, metadata, and whether it is enabled or disabled.
+        This endpoint can help administrators monitor the ammount of quota consumed by tenants.
```
Comment (eskibars) [inline www/static/vectara-oas.yaml:603]: "Ammount"

File: www/static/vectara-oas.yaml [hunk 2]
```diff
@@ -23,6 +23,66 @@ tags:
...
+          Please note that this is an expensive operation, and the requests can be throttled
+
+          by the platform.
+        required: true
+      tags:
+        - AdminService
+      parameters:
+        - $ref: '#/components/parameters/customerIdentifier'
+        - $ref: '#/components/parameters/timeout'
+      description: "Computes the amount of quota consumed across the entire Vectara account. This endpoint is useful for administrators who want to track \nand monitor usage of tenants on their account.\n\nSome tips for this API:\n* This operation only works with OAuth 2.0 (in a JWT \"Bearer Token\") authentication. It does not work with API Keys.  You can find details of how to set up and use OAuth 2.0 [here](https://docs.vectara.com/docs/learn/authentication/oauth-2).\n"
```
Comment (eskibars) [inline www/static/vectara-oas.yaml:57]: Probably best to drop the 2nd sentence or replace it with something else.  This isn't super useful to monitoring usage of *tenants* but rather the account as a whole

**Review by eskibars — CHANGES_REQUESTED** [review]
Thanks for starting this @pwoznic .  See the requested changes

**Review by pwoznic — COMMENTED** [review]
Updated :)

**Comment by eskibars** [comment]
TY!  LGTM

**Comment by eskibars** [comment]
Merging this now, as the blog is going out at 3a PT, and the links would be broken otherwise

**Pull request merged**